src/scripts/sync_synonym.py

```diff
@@ -175,28 +229,67 @@ def sync_synonyms(
         return
 
     # Fetch excluded synonyms
-    mondo_exclusions_df = pd.read_csv(excluded_synonyms_path, sep='\t').fillna('')\
-        .rename(columns=lambda x: x.lstrip('?'))
+    mondo_exclusions_df: pd.DataFrame = _read_sparql_output_tsv(mondo_excluded_synonyms_path)
     mondo_exclusions_df['source_id'] = mondo_exclusions_df.apply(
         lambda row: row['hasDbXref_xref'] if row['hasDbXref_xref'] else row['source_xref'], axis=1)
     mondo_exclusions_df = mondo_exclusions_df.drop(columns=['hasDbXref_xref', 'source_xref'])
     mondo_exclusions_df['synonym_lower'] = mondo_exclusions_df['synonym'].str.lower()
 
     # Query synonyms: source
+    # todo: ideally refactor to use only SPARQL instead of combining SPARQL & OAK results
+    #  - likely quick and easy but there's a chance there's more to it
     source_df: pd.DataFrame = _query_synonyms(mappings_df['source_id'].tolist(), source_db)\
         .rename(columns={'curie': 'source_id'})
     source_df['synonym_lower'] = source_df['synonym'].str.lower()
     source_df['source_label'] = source_df['source_id'].map(source_labels)
+    source_df.drop_duplicates(inplace=True)
```

Comment:
## Bug fix: Dropped some duplicate rows
I believe there were a very small amount of duplicate rows that were making their way into the output, and this fixes that.